src/stylesheets/core/_variables.scss

```diff
@@ -16,6 +16,7 @@ $lead-line-height:    1.7;
 
 $font-sans:           'Source Sans Pro', $helvetica;
 $font-serif:          'Merriweather', $georgia;
+$font-path:           '../fonts'
```

Comment:
should this be:

```
$font-path: '../fonts' !default;
```
so that it can be overridden?

per http://stackoverflow.com/questions/17089717/how-to-overwrite-scss-variables-when-compiling-to-css